client/src/components/Navbar.js

```diff
@@ -31,9 +31,10 @@ export const Navbar = () => {
         {/* <!-- Overlay content --> */}
         <div class="overlay-content">
         <Link to="/">Home</Link>
-        <Link to="/BeginScan">Begin Scan</Link>
         <Link to="/Login">Login</Link>
         <Link to="/SignUp">Sign Up</Link>
```

Comment:
line 34,35: future suggestion to remove this in a following PR and instead link to accounts where you can sign out and stuff